### ISSUE TITLE
E741 false positive

### ISSUE THREAD

**Issue by dannysepler** [PyCQA/pycodestyle#1114]
```python
def a(x=I):
    pass
```

Could probably fix with something similar to https://github.com/PyCQA/pycodestyle/pull/1106

**Comment by asottile** [comment]
#935 is a dupe

**Comment by dannysepler** [comment]
woops! random -- do you think i should just loop in the fix to 935 as part of 1106? probably wouldn't be very much more code 🤷
